src/WebJobs.Script/Rpc/LanguageWorkerConstants.cs

```diff
@@ -10,6 +10,7 @@ public static class LanguageWorkerConstants
         public const int ProcessStartTimeoutSeconds = 60;
         public const string FunctionWorkerRuntimeSettingName = "FUNCTIONS_WORKER_RUNTIME";
         public const string FunctionsWorkerProcessCountSettingName = "FUNCTIONS_WORKER_PROCESS_COUNT";
+        public const string FunctionsNodeVersionSetting = "WEBSITE_NODE_DEFAULT_VERSION";
```

Comment:
Move this to https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

Reply:
After looking through the PR and offline conversation, it does make sense to enable placeholder just for placepholder pools. Please go ahead and remove version matching check. PlaceholderPoolId matching logic maps any major 8 version to ~8 and any major 10 version to ~10.